boards/common/nrf52/Makefile.include

```diff
@@ -18,8 +18,12 @@ else
   PROGRAMMER ?= openocd
 endif
 
-# setup OpenOCD for flashing. Version 0.10 of OpenOCD doesn't contain support
-# for nrf52dk and nrf52840dk boards. To use OpenOCD with these a version
-# build from source (master > 2018, August the 13rd) is required.
+# Setup OpenOCD for flashing. The nRF52 series is supported from OpenOCD v0.11
+# onwards.
```

Comment:
Would you mind updating the docs at https://doc.riot-os.org/group__boards__common__nrf52.html, too?

Reply:
Yes, this is planned for another PR where I address the documentation of the nRF52 boards (see issue #20592).
That upcoming PR will also add the remark in the nRF52840 DK documentation that OpenOCD does not work with newer J-Link firmwares.